cmake/templates/Version.h.in

```diff
@@ -3,6 +3,6 @@
 inline static const std::string GetVersion() {
   // The following line is executed only once on startup or on first call to
   // function.
-  const static std::string Version = "@VERSION_STRING@";
+  const static std::string Version = "2.1.0";
```

Comment:
This change needs to be reverted. With that said, I have had problems with the versioning code (in the *v3.0.0* branch) myself recently so we might have to look into that.
```suggestion
  const static std::string Version = "@VERSION_STRING@";
```